palette/src/lch.rs

```diff
@@ -112,6 +112,32 @@ where
     pub fn from_components<H: Into<LabHue<T>>>((l, chroma, hue): (T, T, H)) -> Self {
         Self::with_wp(l, chroma, hue)
     }
+
+    /// Return the `l` value minimum.
+    pub fn min_l() -> T {
+        T::zero()
+    }
+
+    /// Return the `l` value maximum.
+    pub fn max_l() -> T {
+        from_f64(100.0)
+    }
+
+    /// Return the `chroma` value minimum.
+    pub fn min_chroma() -> T {
+        T::zero()
+    }
+
+    /// Return the `chroma` value maximum. This value will suffice for most
+    /// use cases.
+    pub fn max_chroma() -> T {
+        from_f64(128.0)
+    }
+
+    /// Return the `chroma` extended maximum value.
+    pub fn max_extended_chroma() -> T {
+        from_f64(crate::float::Float::sqrt(127.0f64 * 127.0 + 128.0 * 128.0))
+    }
```

Comment:
The documentation should probably describe the geometric reason for picking one or the other. I don't have any good suggestion off the top of my head right now, but something about how the first one doesn't cover the full space, while still covering enough to be useful, and the second one covering the full space, but also overshoothing.

Oh, and I forgot before, but the square root thingy should maybe use 128 all the way. That way it should reach the negative corner. even if it extends outside all other corners. Once again, this format is weird, but it's the expected one.